.Rprofile

```diff
@@ -1,9 +1,2 @@
 source("renv/activate.R")
 source("r/pre_render_script.R")
```

Comment:
Remove source("r/pre_render_script.R"). There is no reason to call the script by default for every session.

Reply:
I think the idea was to have the same figure specs in your RStudio interactive session as in the render, but I'm ok with deleting it.